File: editorial/chap01/cpp/answer_B03.cpp
#include <iostream>
using namespace std;
 
int main() {
	// 입력
	int N, A[109];
	cin >> N;
	for (int i = 1; i <= N; i++) cin >> A[i];
	
	// 답을 구한다
	bool Answer = false;
	for (int i = 1; i <= N; i++) {
		for (int j = i + 1; j <= N; j++) {
			for (int k = j + 1; k <= N; k++) {
				if (A[i] + A[j] + A[k] == 1000) Answer = true;
			}
		}
	}
	
	// 출력
	if (Answer == true) cout << "Yes" << endl;
	else cout << "No" << endl;
	return 0;
}

File: editorial/chap01/python/answer_B03.py
# 입력
N = int(input())
A = list(map(int, input().split()))

# 답을 구한다
Answer = False
for i in range(N):
	for j in range(i+1, N):
		for k in range(j+1, N):
			if A[i] + A[j] + A[k] == 1000:
				Answer = True

# 출력
if Answer == True:
	print("Yes")
else:
	print("No")
